tuna.go

```diff
@@ -560,3 +562,44 @@ func LoadOrCreateAccount(walletFile, passwordFile string) (*vault.Account, error
 	}
 	return wallet.GetDefaultAccount()
 }
+
+func sendNanoPayment(np *nkn.NanoPay, session *smux.Session, w *nkn.Wallet, inBytes, outBytes, inBytesPaid, outBytesPaid *uint64, c *Common, nanoPayFee string) {
+
+	bytesIn := atomic.LoadUint64(inBytes)
+	bytesOut := atomic.LoadUint64(outBytes)
+
+	delta := c.exitToEntryPrice*common.Fixed64(bytesIn-*inBytesPaid)/TrafficUnit + c.entryToExitPrice*common.Fixed64(bytesOut-*outBytesPaid)/TrafficUnit
```

Comment:
You should use `common.GetPrice()` instead of reading `c.entryToExitPrice` and `c.entryToExitPrice` directly because there is no lock here